core-build/gulp-core-build/src/tasks/jest.schema.json

```diff
@@ -0,0 +1,41 @@
+{
+  "title": "Jest Task configuration",
+  "description": "Defines Jest task configuration",
+
+  "type": "object",
+  "properties": {
+    "$schema": {
+      "description": "Part of the JSON Schema standard, this optional keyword declares the URL of the schema that the file conforms to. Editors may download the schema and use it to perform syntax highlighting.",
+      "type": "string"
+    },
+    "ci": {
+      "description": "Indicate whether the test is running in a CI environment",
+      "type": "boolean"
+    },
+    "configFilePath": {
+      "description": "The path to a jest config file",
+      "type": "string"
+    },
+    "coverage": {
+      "description": "Indicates that test coverage information should be collected and reported in the output",
+      "type": "boolean"
+    },
+    "maxWorkers": {
+      "description": "Specifies the maximum number of workers the worker-pool will spawn for running tests.",
+      "type": "number"
+    },
+    "rootDir": {
+      "description": "The root directory for the project",
+      "type": "string"
+    },
+    "runInBand": {
```

Comment:
>runInBand [](start = 5, length = 9)

Does this mean sync/async?

Reply:
yes, and async has problems in rush.

Reply:
Updated the description.

---
In reply to: [144161174](https://github.com/Microsoft/web-build-tools/pull/391#discussion_r144161174) [](ancestors = 144161174)